_templates/spend-bitcoin.html

```diff
@@ -0,0 +1,19 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+layout: base
+id: spend-bitcoin
+---
+<h1>{% translate pagetitle %}</h1>
+<p class="summary">{% translate summary %}</p>
+
+<h2 id="products-online"><span class="fa fa-shopping-cart fa-lg"></span> {% translate products-online %}</h2>
+<p>{% translate products-online-text %}</p>
+
+<h2 id="directory"><span class="fa fa-book fa-lg"></span> {% translate directory %}</h2>
```

Comment:
As per my other comment, let's remove this section, along with the strings in the other file.